cgsmiles/graph_layout.py

```diff
@@ -199,24 +275,78 @@ def vsepr_layout(self):
                     dist[source][dest] =  d / 2 * math.sqrt(3)
         # do we need this try except????
         try:
-            p0 = nx.fruchterman_reingold_layout(self.graph)
+            p0 = nx.fruchterman_reingold_layout(self.graph, iterations=100)
         except nx.NetworkXException:
             p0 = None
 
         pos = nx.kamada_kawai_layout(self.graph,
                                      pos=p0,
                                      weight=None,
                                      dist=dist,
-                                     scale=len(self.graph) ** 0.5)
+                                     scale=1.0)
```

Comment:
Kamada Kawai and spring are both random layouts. If you want to untangle rings, consider starting with a planar layout (if possible)

Reply:
Kamada-Kawai is not; it minimizes an objective function and the result is largely dependent on the initial guess. By default it starts with a random layout, which is less good for larger molecules. Starting with the springs ensures to disentangle nodes a little.